cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ _Released 03/28/2023 (PENDING)_
 
 **Bugfixes:**
 
+ - Changed the way that we load Git hashes in order to exclude runs for non-relevant commits from the Debug page. Fixes [#26058](https://github.com/cypress-io/cypress/issues/26058).
```

Comment:
nitpick: Should probably go under the Vite fix since that seems more significant